cms/envs/bok_choy_docker.env.json

```diff
@@ -0,0 +1,109 @@
+{
+    "ANALYTICS_SERVER_URL": "",
+    "BOOK_URL": "",
+    "BUGS_EMAIL": "bugs@example.com",
+    "BULK_EMAIL_DEFAULT_FROM_EMAIL": "no-reply@example.com",
+    "CACHES": {
+        "celery": {
+            "BACKEND": "django.core.cache.backends.memcached.MemcachedCache",
+            "KEY_FUNCTION": "util.memcache.safe_key",
+            "KEY_PREFIX": "integration_celery",
+            "LOCATION": [
+                "edx.devstack.memcached:11211"
+            ]
+        },
+        "default": {
+            "BACKEND": "django.core.cache.backends.memcached.MemcachedCache",
+            "KEY_FUNCTION": "util.memcache.safe_key",
+            "KEY_PREFIX": "sandbox_default",
+            "LOCATION": [
+                "edx.devstack.memcached:11211"
+            ]
+        },
+        "general": {
+            "BACKEND": "django.core.cache.backends.memcached.MemcachedCache",
+            "KEY_FUNCTION": "util.memcache.safe_key",
+            "KEY_PREFIX": "sandbox_general",
+            "LOCATION": [
+                "edx.devstack.memcached:11211"
+            ]
+        },
+        "mongo_metadata_inheritance": {
+            "BACKEND": "django.core.cache.backends.memcached.MemcachedCache",
+            "KEY_FUNCTION": "util.memcache.safe_key",
+            "KEY_PREFIX": "integration_mongo_metadata_inheritance",
+            "LOCATION": [
+                "edx.devstack.memcached:11211"
+            ]
+        },
+        "staticfiles": {
+            "BACKEND": "django.core.cache.backends.memcached.MemcachedCache",
+            "KEY_FUNCTION": "util.memcache.safe_key",
+            "KEY_PREFIX": "integration_static_files",
+            "LOCATION": [
+                "edx.devstack.memcached:11211"
+            ]
+        }
+    },
+    "CELERY_ALWAYS_EAGER": true,
+    "CELERY_BROKER_HOSTNAME": "localhost",
+    "CELERY_BROKER_TRANSPORT": "amqp",
+    "CERT_QUEUE": "certificates",
+    "CMS_BASE": "localhost:8031",
```

Comment:
Is this overridden?  Should we comment or use something like ``** OVERRIDDEN **`` as used below?  Same question for other `localhost` references in this file, except for the stubs.